### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by scoobmx**
Add an app that displays Bay Area Rapid Transit train estimated arrivals for a chosen station.  Up to 8 routes can be displayed, for a very dense information density.  Current issues:
1. pixlet lint finds a syntax error on line 22 that I can't locate
2. I'd like to use secret.encrypt but do not know the syntax
3. I couldn't test the schema since I don't have tidbyt plus

### CODE REVIEW

**Comment by scoobmx** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/denser_bart/denser_bart.star [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+PREDICTIONS_URL = "https://api.bart.gov/api/etd.aspx"
+STATIONS_URL = "https://api.bart.gov/api/stn.aspx"
+DEFAULT_ABBR = "WOAK"
+DEFAULT_KEY = "MW9S-E7SL-26DU-VV8V"
+
+def main(config):
+    abbr = config.get("abbr")
+    if abbr == None:
+	abbr = DEFAULT_ABBR
```
Comment (matslina) [inline apps/denser_bart/denser_bart.star:22]:
```suggestion
        abbr = DEFAULT_ABBR
```

Comment (matslina) [reply apps/denser_bart/denser_bart.star:22]: tabs vs spaces

File: apps/denser_bart/denser_bart.star [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    
+    if num_routes == 0:
+        return render.Root(
+            child = render.Box(
+                child = render.Text("No Data", font = "6x13", color = "#fff"),
+            ),
+        )
+    elif num_routes < 5:
+        # Show a single column
+	train_rows = []
```
Comment (matslina) [inline apps/denser_bart/denser_bart.star:35]:
```suggestion
        train_rows = []
```

**Comment by scoobmx** [comment]
Hi @matslina what is the proper syntax for pixlet encrypt, specifically order of arguments, how to g=get and supply app ID, and whether string arguments need a ""?

**Comment by matslina** [comment]
`pixlet encrypt [app ID] [secret value]`

You have the app ID in `manifest.yaml`. Note that `secret.decrypt()` will return `None` when running it locally with pixlet. To get through CI, you'll need to handle that case gracefully, otherwise the tests will fail and we won't be able to merge the PR.

**Pull request merged**